tutorminio/patches/openedx-common-settings

```diff
@@ -3,6 +3,10 @@ VIDEO_IMAGE_SETTINGS["STORAGE_KWARGS"]["location"] = VIDEO_IMAGE_SETTINGS["STORA
 VIDEO_TRANSCRIPTS_SETTINGS["STORAGE_KWARGS"]["location"] = VIDEO_TRANSCRIPTS_SETTINGS["STORAGE_KWARGS"]["location"].lstrip("/")
 GRADES_DOWNLOAD["STORAGE_KWARGS"] = {"location": GRADES_DOWNLOAD["STORAGE_KWARGS"]["location"].lstrip("/")}
 GRADES_DOWNLOAD["STORAGE_KWARGS"]["bucket_name"] = "{{ MINIO_GRADES_BUCKET_NAME }}"
+OPENEDX_LEARNING["MEDIA"]["BACKEND"] = DEFAULT_FILE_STORAGE
+OPENEDX_LEARNING["MEDIA"]["OPTIONS"] = {
+    'bucket_name': "{{ MINIO_OPENEDX_LEARNING_BUCKET_NAME }}",
```

Comment:
Does there need to be anything to force this bucket to be private? Or does that happen by default?

Reply:
buckets are by-default private in minio, we do have the policy to make the bucket public.

Reply:
Okay, great. To be clear: we definitely want it to be private. Thank you.

Reply:
<img width="1785" alt="Screenshot 2024-10-31 at 11 24 02 AM" src="https://github.com/user-attachments/assets/ad6b4740-4bd0-4842-8472-47c74d9f6abc">